frontend/src/components/config/GoogleLogin.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+import firebase from 'firebase';
+import app from '../../api/Firebase';
+import { PrimaryButton } from '../core/Button';
+
+const GoogleButton = styled(PrimaryButton)`
+  position: relative;
+  display: block;
+  margin: 10px auto;
+  background: ${({ theme }) => theme.colors.white};
+  text-align: center;
+`;
+
+const GoogleImage = styled.img`
+  display: inline;
+  position: absolute;
+  left: 15px;
+  width: 30px;
+  vertical-align: middle;
```

Comment:
```suggestion
  position: absolute;
  left: 15px;
  width: 30px;
```
Remove those attributes because I don't think they actually do anything here.